taipy/config/__init__.py

```diff
@@ -105,3 +105,9 @@ def _inject_section(
 
     for exposed_configuration_method, configuration_method in configuration_methods:
         setattr(Config, exposed_configuration_method, configuration_method)
+
+
+@_config_doc
+def _inject_method(configuration_methods: List[tuple]):
+    for exposed_configuration_method, configuration_method in configuration_methods:
+        setattr(Config, exposed_configuration_method, configuration_method)
```

Comment:
I understand that it's for Databrick.

Can you give an example how can this method being used?

Reply:
yep sure thing, so we want to add a `configure_databricks_task` method to the Config, only the method. But the inject_section config also expects the section and many other things which don't relate to what we want to do with a single method. And so i added the function :D